### DISCUSSION TITLE
Does .git folder track pull request comments, pull request descriptions?

### DISCUSSION THREAD

**Discussion by JQGoh** [community/community#112586] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I notice that the .git folder is large and that is due to the.pack file.

My understanding is that .git folder does not track information related to pull request descriptions, said I attach screenshots into the PR, or reviewers' comments.

Could anyone of you help to verify this?

**Comment by ghost** [comment]
Hi @JQGoh 

No, the .git folder in a Git repository does not directly track pull request comments or descriptions.

The .git folder contains all the internal data and metadata that Git uses to manage the repository, such as commits, branches, tags, and configuration settings. It doesn't directly interact with external systems like GitHub or GitLab, where pull requests are managed.

Pull request comments, descriptions, and other metadata specific to a platform like GitHub or GitLab are typically stored within the platform's own database and are not part of the .git folder or the Git repository itself. These platforms often provide APIs that allow you to interact with this metadata programmatically.

**Reply by JQGoh** [reply]
@KohoolateSen Thanks for confirming this

**Reply by ghost** [reply, marked as answer]
Your welcome ! 
if you got the answer please mark this discussion as answered 
Good luck :)
